x/incentives/keeper/keeper_test.go

```diff
@@ -45,3 +46,18 @@ func (s *KeeperTestSuite) ValidateDistributedGauge(gaugeID uint64, expectedFille
 func (s *KeeperTestSuite) ValidateNotDistributedGauge(gaugeID uint64) {
 	s.ValidateDistributedGauge(gaugeID, 0, sdk.Coins(nil))
 }
+
+func (s *KeeperTestSuite) ValidateIncentiveRecord(poolId uint64, remainingCoin sdk.DecCoin, emissionRate sdk.Dec, incentiveRecord cltypes.IncentiveRecord) {
+	s.Require().Equal(poolId, incentiveRecord.PoolId)
+	s.Require().Equal(emissionRate, incentiveRecord.GetIncentiveRecordBody().EmissionRate)
+	s.Require().Equal(time.Nanosecond, incentiveRecord.MinUptime)
+	s.Require().Equal(remainingCoin, incentiveRecord.GetIncentiveRecordBody().RemainingCoin)
+}
+
+func (s *KeeperTestSuite) ConvertCoinsToDecCoins(coins sdk.Coins) sdk.DecCoins {
+	decCoins := sdk.DecCoins{}
+	for _, coin := range coins {
+		decCoins = append(decCoins, sdk.NewDecCoin(coin.Denom, coin.Amount))
+	}
+	return decCoins
+}
```

Comment:
This could probably live in either osmoutils or the high level keeper test suite.